doc/Pacemaker_Explained/en-US/Ch-Constraints.txt

```diff
@@ -250,7 +250,7 @@ alternatively, a group (see <<group-resources>>).
 === Ordering Properties ===
 
 .Properties of a rsc_order Constraint
-[width="95%",cols="1m,1,4<a",options="header",align="center"]
+[width="95%",cols="1m,1,<4",options="header",align="center"]
```

Comment:
Does this affect the bulleted list in the kind description?

Reply:
Yes, it does, and thanks for pointing that out.
It suffices to mark the affected cells as `asciidoc`, though.
Hidden comments ditto.

Reply:
It also means that anchors cannot likely be placed into lists withing
the table cells, which may be the most probable reason of placing
the anchor at all :-/